src/theme.js

```diff
@@ -48,6 +49,8 @@ const theme = {
     x5l: '2.25rem',
     x6l: '3rem',
     x7l: '4rem',
+    x8l: '5rem',
```

Comment:
You are not using `x8l` anywhere, just `x9l`. Maybe you should change the `x8l` value to be 6 rem and then remove `x9l`. 
Besides that, I've been taking a look at all the styles files and I couldn't find any component using `x7l`, `x6l`, and `x5l` values. 
I believe those values were added in the `Navbar` styles migration, which was one of the first ones, so it involved a significant part of the `theme.js` setup. @Rminetti do you know if there is a reason to still keep them there? If it isn't we should double-check because I might be missing something.

Reply:
if they are not being used I don't see why we should keep them.

Reply:
I don't have a strong opinion on what I said earlier, and I agree that the font size jump would be weird.
Im not entirely sure about this `I see the theme.js file as a palette of styles, and the user can select styles from there, but it doesn't mean that they have to use ALL of them`. 
If i think about it I would say that in some properties like colors or box shadows, the theme should define what gets used on the app and not the other way around. But on the case of the font size I agree that is harmless to leave them (In my opinion)

Reply:
Regarding @Rminetti last comment, in my opinion, is the same for colors and font sizes, you should have them all defined from the get-go in an ideal situation. Given that this is not designed properly I would add them as need dictates.